src/feature/stats/bw_array.h

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2007-2020, The Tor Project, Inc. */
+/* See LICENSE for licensing information */
+
+/**
+ * @file bw_array_st.h
+ * @brief Bandwidth array manipulation functions.
+ **/
+
+#ifndef BW_ARRAY_H
+#define BW_ARRAY_H
+
+#include "feature/stats/bw_array_st.h"
+
+STATIC void commit_max(bw_array_t *b);
```

Comment:
The `STATIC` methods should be protected by an `BW_ARRAY_PRIVATE` guard, such that people don't accidentally use the static functions in the wrong places. Have a look at `src/lib/geoip/geoip.h` for an example of how it is done elsewhere.

Reply:
This entire header is a private API, as is bw_array_st.h. They should both be protected by _PRIVATE guard macros.

That way, they won't be used outside the rephist module.

Reply:
Thanks for the suggestion. Made the contents of both the files private.